string pos_path
string retry_path
string output_path
string[] neg_path
float32[] start_gps